lib/tasks/update_is_partner_for_code_schools.rake

```diff
@@ -0,0 +1,8 @@
+namespace :is_partner do
+  desc "Updates column is_partner for code schools"
+  task code_schools: :environment do
```

Comment:
The `rake` task would be similar to the one you did for community leaders.  In the sense that you'll need to figure out which code schools, in prod, need to have this set to `is_partner: true`.

And then we would update just those specific schools.

To get started on that list, perhaps connect with @ksmacleod99 .  She might be able to point you in the right direction.